### PR TITLE
use ibc-hd-signer

### PR DESCRIPTION
**Pull request by dai1975**
edit go.mod after merging 
-[ ethereum-ibc-relay-chain](https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/46)
-[ ibc-hd-signer](https://github.com/datachainlab/ibc-hd-signer/pull/1)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,12 +5,14 @@ go 1.21
...
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	github.com/datachainlab/ibc-hd-signer v0.0.0 => github.com/datachainlab/ibc-hd-signer v0.0.0-20240702051257-f69056d48080
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
 )
 
 require (
-	github.com/datachainlab/ethereum-ibc-relay-chain v0.3.2
-	github.com/hyperledger-labs/yui-relayer v0.5.3
+	github.com/datachainlab/ethereum-ibc-relay-chain v0.3.5
+	github.com/datachainlab/ibc-hd-signer v0.0.0
```
Comment (siburu) [inline go.mod]: Use v0.1.0 as a version: https://github.com/datachainlab/ibc-hd-signer/releases/tag/v0.1.0

**Review by siburu — COMMENTED** [review]
@dai1975 Thank you. Just one point to fix.

File: go.mod (hunk 1 above)
Comment (dai1975) [reply go.mod]: fixed

**Review by siburu — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**